tests/testthat/test-derive_joined.R

```diff
@@ -242,3 +242,66 @@ test_that("derive_vars_joined Test 7: new_vars expressions using variables from
     keys = c("USUBJID", "AESEQ")
   )
 })
+## Test 8: NULL new_vars will still remove .join columns ----
+test_that("derive_vars_joined Test 8: NULL new_vars will still remove .join columns", {
```

Comment:
Should we update the title to something like "error if new_vars are already in dataset"?